README.md

```diff
@@ -34,25 +34,27 @@ Use root or package level commands to run specific package, e.g. web:
 
 [/packages/web] yarn start
 ```
+You should add your `google-services.json` from firebase configs to '[/packages/mobile/android/app].
 Use root or package level commands to run mobile:
 ```
-[/] yarn workspace mobile start:android
+[/] yarn workspace mobile android
 
 [/packages/mobile] yarn start:android
```

Comment:
If you change the above command change this one too.

P.S. Both comments apply to the ios part below.

Reply:
Thanks :)